src/services/services.ts

```diff
@@ -5724,6 +5696,23 @@ module ts {
         return { getClassificationsForLine };
     }
 
+    /// getDefaultLibraryFilePath
+    declare var __dirname: string;
+    declare var module: any;
+
+    /**
+      * Get the path of the default library file (lib.d.ts) as distributed with the typescript
+      * node package.
+      * The functionality is not supported if the ts module is consumed outside of a node module. 
+      */
+    export function getDefaultLibraryFilePath(options: CompilerOptions): string {
+        if (typeof module !== "undefined" && module.exports) {

```

Comment:
checking if we are in node. i am open to suggestions.. this is how we do it in sys anyways.